docs/vpc-configuration.md

```diff
@@ -0,0 +1,159 @@
+# VPC Configuration
+
+### Contents
+
+##### [1. Basic configuration](#basic-configuration)
+
+VPC configuration is done in the under the 'vpc' key section of the cloudformation configuration file.
+
+Defaults are set so that without specifying anything, we get a VPC with CIDR `10.128.0.0/16` and 3 subnets, `10.0.0.0/20`, `10.0.16.0/20`, and `10.0.32.0/20`
```

Comment:
The 3 /16 subnets do not fall within the /20 . (if there's a reason for this then it's not clear to me)